utilities/update_stdlib_pkgimage_checksums.jl

```diff
@@ -0,0 +1,99 @@
+using Libdl
+import Base: _sizeof_uv_fs, uv_error
+
+# This should be in Julia base, I feel
```

Comment:
Yeah this should be in contrib, so that we update it there when the header format changes

Reply:
Sounds good. Perhaps could be a follow-on PR?